project1/application.py

```diff
@@ -24,3 +24,16 @@
 @app.route("/")
 def index():
     return "Project 1: TODO"
```

Comment:
Add a welcome page instead of sending text.

Reply:
i have now added a page index.html.
Is it ok now?